client/deposits/details/index.tsx

```diff
@@ -228,6 +233,53 @@ export const DepositOverview: React.FC< DepositOverviewProps > = ( {
 					] }
 				</SummaryList>
 			) }
+			<Card>
+				<CardHeader>
+					<Text size={ 16 } weight={ 600 }>
+						{ isWithdrawal
+							? __( 'Withdrawal details', 'woocommerce-payments' )
+							: __( 'Payout details', 'woocommerce-payments' ) }
+					</Text>
+				</CardHeader>
+				<CardBody>
+					<div className="woopayments-payout-details-header">
+						<h2>
+							{ __( 'Bank account', 'woocommerce-payments' ) }
+						</h2>
+						<div className="woopayments-payout-details-header__value">
+							{ deposit.bankAccount }
+						</div>
+						<h2>
+							{ __(
+								'Bank reference ID',
+								'woocommerce-payments'
+							) }
+						</h2>
+						<div>
+							{ deposit.bank_reference_key ? (
+								<>
+									<span className="woopayments-payout-details-header__bank-reference-id">
+										{ deposit.bank_reference_key }
+									</span>
+									<CopyButton
+										textToCopy={
+											deposit.bank_reference_key
+										}
+										label={ __(
+											'Copy bank reference ID to clipboard',
+											'woocommerce-payments'
+										) }
+									/>
+								</>
+							) : (
+								<div className="woopayments-payout-details-header__value">
+									Not available
```

Comment:
We should allow this string to be translatable.
```suggestion
									{ __(
										'Not available',
										'woocommerce-payments'
									) }
```

Reply:
Thanks for catching this! I have added the translation tag here - [24dc671](https://github.com/Automattic/woocommerce-payments/pull/9945/commits/24dc6711f9a6a2940e11f36eeb89dfef360b9538)